backend/core/serializers.py

```
@@ -303,7 +303,7 @@ class AssetWriteSerializer(BaseModelSerializer):
 
     class Meta:
         model = Asset
```

Ruff (0.8.2)

`Asset` may be undefined, or defined from star imports

(F405)